spec/features/visitors/view_bulletings_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# Feature: Bulletins page
+#   So that I can understand potential health dangers in my area
+#   As a visitor who has chosen a particular zone
+#   I want to see warnings and alerts for both pollutants and non-pollutants for 3 days
+RSpec.feature "Bulletins page" do
+  # Scenario: Visit the home page
+  #   Given I am a visitor
+  #   When I visit the home page
+  #   And I select "View bulletins"
+  #   Then I see the bulletins page
+  #   And I see local air quality information
+  scenario "visit the bulletins page" do
+    visit root_path
+    when_i_select_view_bulletins
+    then_i_see_the_bulletins_page
+    and_i_see_local_air_quality_information
+  end
+
+  def when_i_select_view_bulletins
+    click_link("View bulletins")
+  end
+
+  def then_i_see_the_bulletins_page
+    expect(page).to have_content("Bulletins")
+  end
+
+  def and_i_see_local_air_quality_information
+    expect(page).to have_content("Three day forecast for Haringey")
+    within first(".govuk-table__row") do
+      page.all("td").each_with_index do |td, index|
+        if index == 0
+          expect(td).to have_content("Air pollution")
+        else
+          expect(td).to have_content("Low")
+        end
```

Comment:
I expect that in coming PRs we'll be extending this considerably to describe the range of content which is shown for various results in the categories of:

- Air pollution
- UV
- Pollen
- Temperature

Reply:
yes, absolutely